utils/general.py

```diff
@@ -3,10 +3,11 @@
 SPDX-License-Identifier: BSD-2-Clause
 '''
 import os
+import random
 import re
 
 from contextlib import contextmanager
-
```

Comment:
If you could keep this line to separate the internal dependencies from the external dependencies, I'd appreciate it.